pkg/network/prometheus.go

```diff
@@ -34,6 +35,14 @@ var (
 		[]string{"description", "value"},
 	)
 
+	neogoVersion = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Help:      "NeoGo version and network server ID",
+			Name:      "neogo_version",
+			Namespace: "neogo",
+		},
+		[]string{"version", "server_id"})
```

Comment:
Do we want to mix `server_id` in? It's a separate thing to me.

Reply:
Fixed.